clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeployManifestIT.java

```diff
@@ -1246,4 +1247,199 @@ public void shouldBindRequiredCronJobDockerImage() throws IOException, Interrupt
         imageDeployed,
         "Expected correct " + DEPLOYMENT_1_NAME + " image to be scheduled");
   }
+
+  @DisplayName(
+      ".\n===\n"
+          + "Given a v1beta1 CRD manifest\n"
+          + "When sending deploy manifest request\n"
+          + "Then a v1beta1 CRD is created\n===")
+  @Test
+  public void shouldDeployCrdV1beta1() throws IOException, InterruptedException {
+    // ------------------------- given --------------------------
+    final String crdName = "crontabs.stable.example.com";
+    final List<Map<String, Object>> manifest =
+        KubeTestUtils.loadYaml("classpath:manifests/crd_v1beta1.yml")
+            .withValue("metadata.name", crdName)
+            .asList();
+    // ------------------------- when --------------------------
+    final List<Map<String, Object>> request =
+        KubeTestUtils.loadJson("classpath:requests/deploy_manifest.json")
+            .withValue("deployManifest.account", ACCOUNT1_NAME)
+            .withValue("deployManifest.moniker.app", APP1_NAME)
+            .withValue("deployManifest.manifests", manifest)
+            .asList();
+    final List<String> deployments = KubeTestUtils.sendOperation(baseUrl(), request, "");
```

Comment:
Instead of `KubeTestUtils.sendOperation` you can use `KubeTestUtils.deployAndWaitStable` like the other test cases. It does a `GET /manifest` request to check if the CRD was deployed successfully, which is what orca uses when running a pipeline.